audio_program_generator/apg.py

```diff
@@ -186,8 +202,11 @@ def main():
     sound_file = Path(args["<sound_file>"]) if args["<sound_file>"] else None
     slow = bool(args["--slow"])
     attenuation = int(args["--attenuation"]) if args["--attenuation"] else 0
+    tld = str(args["--tld"]) if args["--tld"] else "com"
+    tld = tld if tld in TLDs else "com"
```

Comment:
I usually default to a `set` for membership checking ( and if order does not matter)